posts/2024-02-26-2023-Rust-Annual-Survey-2023-results.md

```diff
@@ -0,0 +1,385 @@
+</noscript></div>
+    <div style="display: flex; margin-bottom: 10px;">
+        <span>[<a href="../../../images/2024-02-rust-survey-2023/what-are-your-preferred-languages-for-technical-communication.png" target="_href_" title="Download chart as PNG">PNG</a>]</span>&nbsp;<span>[<a href="../../../images/2024-02-rust-survey-2023/what-are-your-preferred-languages-for-technical-communication.svg" target="_href_" title="Download chart as SVG">SVG</a>]</span>
+    </div>
+</div>
+<!-- Chart what-are-your-preferred-languages-for-technical-communication end -->
+
+We also asked whether respondents consider themselves members of a marginalized community. Out of those who answered, 76% selected no, 14% selected yes, and 10% preferred not to say.
+
+We have asked the group that selected “yes” which specific groups they identified as being a member of. The majority of those who consider themselves a member of an underrepresented or marginalized group in technology identify as lesbian, gay, bisexual, or otherwise non-heterosexual. The second most selected option was neurodivergent at 41% followed by trans at 31.4%. Going forward, it will be important for us to track these figures over time to learn how our community changes and to identify the gaps we need to fill.
+
+<!-- Chart which-marginalized-group start -->
+<div>
+    <div class="bar-chart" id="which-marginalized-group" style="height:500px; width:100%;"><noscript>
+<img alt="which-marginalized-group" height="500" src="../../../images/2024-02-rust-survey-2023/which-marginalized-group.png"/>
+</noscript></div>
+    <div style="display: flex; margin-bottom: 10px;">
+        <span>[<a href="../../../images/2024-02-rust-survey-2023/which-marginalized-group.png" target="_href_" title="Download chart as PNG">PNG</a>]</span>&nbsp;<span>[<a href="../../../images/2024-02-rust-survey-2023/which-marginalized-group.svg" target="_href_" title="Download chart as SVG">SVG</a>]</span>
+    </div>
+</div>
+<!-- Chart which-marginalized-group end -->
+
+As Rust continues to grow, we must acknowledge the diversity, equity, and inclusivity (DEI)-related gaps that exist in the Rust community. Sadly, Rust is not unique in this regard. For instance, only 20% of 2023 respondents to this representation question consider themselves a member of a racial or ethnic minority and only 26% identify as a woman. We would like to see more equitable figures in these and other categories. In 2023, the Rust Foundation formed a diversity, equity, and inclusion subcommittee on its Board of Directors whose members are aware of these results and are actively discussing ways to better support underrepresented groups and also help ensure these numbers improve. We look forward to learning more about the DEI-related initiatives the Rust Foundation will help support. This data is a helpful place to start. [a]
```

Comment:
Is this link meant as a footnote? If yes, then the markdown incantation is as follows:

```suggestion
As Rust continues to grow, we must acknowledge the diversity, equity, and inclusivity (DEI)-related gaps that exist in the Rust community. Sadly, Rust is not unique in this regard. For instance, only 20% of 2023 respondents to this representation question consider themselves a member of a racial or ethnic minority and only 26% identify as a woman. We would like to see more equitable figures in these and other categories. In 2023, the Rust Foundation formed a diversity, equity, and inclusion subcommittee on its Board of Directors whose members are aware of these results and are actively discussing ways to better support underrepresented groups and also help ensure these numbers improve. We look forward to learning more about the DEI-related initiatives the Rust Foundation will help support. This data is a helpful place to start[^1].
```